docs/content/paket-and-dotnet-cli.md

```diff
@@ -40,25 +46,46 @@ This step is absolutely the same as with traditional .NET projects.
 
 Like with traditional .NET projects you also need to put a [`paket.references` files](references-files.html) alongside your MSBuild project files.
 
+After `paket.references` files are created, to update your existing projects, just run
+
+```sh
+.paket/paket.exe install
```

Comment:
part of new workflow is that you don't need to run install after a references file change. everything now happens at "dotnet restore". So you need to run that instead